src/backend/marsha/core/migrations/0016_auto_20191001_0812.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.5 on 2019-10-01 08:12
+
+import json
+import re
+
+from django.conf import settings
+from django.db import migrations
+
+import boto3
+
+
+def escape_timed_text_tracks(apps, schema_editor):
+    if not settings.SHOULD_RUN_TIMEDTEXT_TRACKS_ESCAPING_MIGRATION:
```

Comment:
add explanations in docstring?

Reply:
@lunika still no docstring ;-)

Reply:
docstring about what ? The if statement ?

Reply:
no by docstring I mean the introductory paragraph of the function like you did on the reverse migration.

Reply:
Done